app/policies/user_policy.rb

```diff
@@ -22,27 +22,65 @@ def index?
   def edit_by_project_admin?
     project_admin? && record.portal_teacher
   end
```

Comment:
I'm not sure where else this being used. But in the case of the view in this PR, the project admin should also be able to edit and update students that are associated with their projects.